main.c

```diff
@@ -27,27 +38,72 @@ static void LED_init() {
     LATC7 = 1;      // turn the led off
 }
 
+void set_DACs(){
```

Comment:
It would make sense to put this in lin_actuator.c since it's directly related to that functionality.